### PR TITLE
fix universal link when no item content obj passed

### PR DESCRIPTION
**Pull request by nileshgulia1**

### CODE REVIEW

**Comment by tiberiuichim** [comment]
@nileshgulia1 A more accurate changelog entry is needed. A testcase might be nice to have as well

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@
 
 ### Bugfix
 
+- fix universal link @nileshgulia1s
```
Comment (pnicolli) [inline CHANGELOG.md:13]:
```suggestion
- fix universal link when no item content obj passed @nileshgulia1
```
I would add a bit more explanation here, just to make it clearer. The PR title is fine imho

**Review by pnicolli — COMMENTED** [review]
I'll add this as a comment review since there's just really a non-blocking minor suggestion I made in the comments.
Apart from my comment on the changelog, the PR is good to go.
Another way to improve this component, imho, would be to add ``.isRequired`` to the ``'@id': PropTypes.string,`` field of the item prop, because we want that in case the ``item`` prop is passed.

File: CHANGELOG.md (hunk 1 above)
Comment (nileshgulia1) [reply CHANGELOG.md:13]: I fixed the typo in changelog and made it more accurate. Also added `isRequired` propType to `item[@id]`

**Review by pnicolli — APPROVED** [review]

**Comment by nileshgulia1** [comment]
@sneridagh We can also merge this?

**Review by tiberiuichim — APPROVED** [review]

**Review by sneridagh — APPROVED** [review]

**Pull request merged**